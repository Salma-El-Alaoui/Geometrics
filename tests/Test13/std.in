C c1 1 1 1
R r1 1 1 1 1
L l1 1 1 1 1
OA oa1 c1 r1
OA oa2 r1 l1
OA oa oa1 oa2
MOVE oa 1 1
LIST
EXIT
