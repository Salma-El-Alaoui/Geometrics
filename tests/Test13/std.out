OK
#Element ajoute: c1
OK
#Element ajoute: r1
OK
#Element ajoute: l1
OK
#Element ajoute: oa1
OK
#Element ajoute: oa2
OK
#Element ajoute: oa
OK
#oa deplace
C c1 2 2 1
L l1 2 2 2 2
OA oa oa1 oa2
OA oa1 c1 r1
OA oa2 l1 r1
R r1 2 2 2 2
